x/swaprouter/module/module.go

```diff
@@ -142,7 +143,7 @@ func (am AppModule) SimulatorGenesisState(simState *module.SimulationState, s *s
 	swaprouterGen := types.DefaultGenesis()
 	// change the pool creation fee denom from uosmo to stake
 	// TODO: uncomment this once simulation is enabled.
-	// swaprouterGen.Params.PoolCreationFee = sdk.NewCoins(swaproutersimulation.PoolCreationFee)
+	swaprouterGen.Params.PoolCreationFee = sdk.NewCoins(gammsimulation.PoolCreationFee)
```

Comment:
Shouldn't this now read "swaproutersimulation.PoolCreationFee" since we are no longer reading it from gamm? If so, we should remove whatever var or const this is pointing to from gamm right?

Also, the above TODO can be removed now I believe.

Reply:
Removing TODO.

I will add package change task as a TODO in: https://github.com/osmosis-labs/osmosis/issues/3556

Currently, there is no swaprouter simulation package yet